packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1372,6 +1373,7 @@ function mountIndeterminateComponent(
       context,
       renderExpirationTime,
     );
+    setIsRendering(false);
```

Comment:
If the component throws, `setIsRendering` won't be reset. This needs to be reset in the the work loop. Here: https://github.com/facebook/react/blob/8311cb5d24fd1648ddffb659bf9b3317f5f885d0/packages/react-reconciler/src/ReactFiberWorkLoop.js#L1226-L1229

Reply:
Never mind, gets reset by `resetCurrentDebugFiberInDEV